frontends/precinct-scanner/src/screens/setup_scanner_screen.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
+import { fontSizeTheme, Main, MainChild, Prose, Screen } from '@votingworks/ui';
+
+function doNothing() {
+  // do nothing
+}
+
+interface Props {
+  useEffectToggleLargeDisplay?: () => void;
```

Comment:
this is a VxMark thing I think so you can remove it

Reply:
thanks!